portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path

```diff
@@ -90,6 +90,16 @@
 	<td>//div[contains(text(),'${columnName}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>COLUMN_RESIZER</td>
+	<td>//*[@class="dnd-th-resizer is-allowed"]</td>
```

Comment:
All columns share the same path, and since the ID column is the first, I think it's safer if we add something like `contains(text('${key_column}'))` to it in order to make sure the column we want is to be resized.

Reply:
sounds like a good improvement